src/components/NewNursery/index.tsx

```diff
@@ -122,6 +160,23 @@ export default function NurseryView(): JSX.Element {
                 errorText={record.description ? '' : descriptionError}
               />
             </Grid>
+            {timeZoneFeatureEnabled && (
+              <Grid item xs={gridSize()}>
+                <TimeZoneSelector
+                  selectedTimeZone={record?.timeZone || defaultTimeZone.id}
+                  onTimeZoneSelected={onChangeTimeZone}
+                  label={strings.TIME_ZONE}
+                  disabled={orgTZChecked}
+                />
+                <Checkbox
```

Comment:
since the dropdown blocks the checkbox.. should we move this to the right (or i wish between label and dropdown but i guess that's not feasible)... just a thought. or we can let design make a decision

Reply:
I think we can update it later with design decision

Reply:
sounds good